### DISCUSSION TITLE
Download repository button embedded on my website

### DISCUSSION THREAD

**Discussion by icelify** [community/community#22758] [New to GitHub]
<p>My idea is to create two buttons on my website page about a project posted on github.<br>
First - a standard button linking to github respository.<br>
Another one named - download from github - that would be actually downloading repository on one click, to make it faster and easier for some users.<br>
Although I can’t find any information if this is against github terms of service or not.<br>
So please tell me, if this is fine.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Generally speaking, it should be fine to create buttons that link to a GitHub repository and to a direct download of the repository’s content.</p>
<p>Please note, however, that GitHub is not in a position to prescreen content and as such cannot review whether your product or actions would violate any of our policies or guidelines.</p>
<p>Should we receive a report about your content, we will review it at that time to determine if a violation has occurred.</p>

**Comment by icelify** [comment]
<p>Thank you for your answer. If it should be fine to create a direct download button of repository as well as linking to that repository next to it, it should not be any problem with violation policies than.</p>
